### DISCUSSION TITLE
Costomization of Doks

### DISCUSSION THREAD

**Discussion by technosprout** [thuliteio/doks#1126] [Q&A]
Doks is good out of the box. However, I need to customize the site for a project.
CSS changes, changes to the layouts etc. What is the official way to do this? The new updates have removed the "my-doks-theme"

**Comment by h-enk** [comment]
> The new updates have removed the "my-doks-theme"

All subdirectories from "my-doks-theme" are now in the root of your project

Customize SCSS: 

- `assets/scss/common/_custom.scss`
- `assets/scss/common/_variables-custom.scss`

Customize layout files: 

- Copy the file to override from `node_modules/@hyas/doks-core/layouts/` to `layouts/` — keep the folder structure the same

**Reply by technosprout** [reply]
Thank you
